client/settings/deposits/index.js

```diff
@@ -238,10 +238,7 @@ const Deposits = () => {
 
 				<div className="deposits__bank-information">
 					<h4>
-						{ __(
-							'Bank account information',
-							'woocommerce-payments'
-						) }
+						{ __( 'Deposit bank account', 'woocommerce-payments' ) }
```

Comment:
Just wanted to verify this text change is intentional, as it doesn't seem to be related to the rest of the changes here.

Reply:
Yep. It's in the Figma files. So I guess it would be ok to also update this.